lib/src/components/pagination/pagination.constants.ts

```diff
@@ -4,3 +4,7 @@ export enum VisibleElementsAmount {
   LESS = 6,
   MORE = 8
 }
+
+export const MAX_PAGINATION_ITEMS = 4
```

Comment:
how come these arent 5 and 3 respectively?

Reply:
Because the 1st and last pagination items are rendered separately from this list, so we only want to return 4 or 2 as the 5th and 3rd items are separate.

Reply:
This is one thing I also find confusing tbf.

Reply:
it could be just a comment explaining this. 
```suggestion
export const MAX_PAGINATION_ITEMS = 4 // excluding first and last pagination items that are always rendered
```

Reply:
Yeah that's fair, I didn't want to rock the boat too much by changing these values but a comment could help